ui/src/components/footer/LinkGrid.js

```diff
@@ -0,0 +1,46 @@
+import { Box, Link, SimpleGrid, Stack, Text } from "@chakra-ui/react";
+import * as React from "react";
+
+export function LinkGrid() {
+  return (
+    <SimpleGrid
+      columns={[2, 2, 4]}
+      textAlign="left"
+      minW="50%"
+      spacing={["2vh", "2vh", "2vw"]}
+    >
+      <Box>
+        <Stack>
+          <Text fontWeight="bold">Visit</Text>
+          <Link>Church online</Link>
+        </Stack>
+      </Box>
+      <Box>
+        <Stack>
+          <Text fontWeight="bold">Connect</Text>
+          <Link>Ministries</Link>
```

Comment:
these links should link to actual existing pages if they are already implemented.

Reply:
connected to the pages on our old website